tock/tock/static/sass/core.scss

```diff
@@ -17,7 +15,7 @@ $theme-font-path:          "../../vendor/uswds_v2.3.1/fonts";
   position: absolute; // for some reason, USWDS 2.x overrides this to set position as relative, which breaks nav
 }
 
-
+//test comment --lg 11
```

Comment:
**question:** Is this actually used in a test case or should it be taken out to keep things clean?